app/assets/javascripts/app/views.js

```diff
@@ -65,7 +65,8 @@ app.views.Base = Backbone.View.extend({
     var self = this;
     _.each(this.subviews, function(property, selector){
       var view = _.isFunction(self[property]) ? self[property]() : self[property];
-      if(view) {
+      if (view && self.$(selector).length > 0) {
+        self.$(selector).first().empty();
```

Comment:
Unfortunately I don't remember the exact reason I added this line. However, I checked it right now and I see it changes the behavior on the getting started page. I'll study what's going on here and report.